examples/asr/transcribe_speech.py

```diff
@@ -340,14 +340,19 @@ def autocast(dtype=None):
                     decoder_type=cfg.decoder_type,
                 )
             else:
-                transcriptions = asr_model.transcribe(
-                    paths2audio_files=filepaths,
-                    batch_size=cfg.batch_size,
-                    num_workers=cfg.num_workers,
-                    return_hypotheses=cfg.return_hypotheses,
-                    channel_selector=cfg.channel_selector,
-                    augmentor=augmentor,
-                )
+                for _ in range(5):
```

Comment:
I agree that we should measure speed after warmup, but 5x transcriptions is too much by default.

I suggest to do something like:
- add `warmup_batches=0` flag to config
- warmup with `warmup_batches` batches (from `filepaths`)
- transcribe as usual

By default user will just get a transcription, but for measuring speed we can use e.g., `warmup_batches=10`.

Reply:
I like this idea but actually let's not put benchmarking code inside a user facing inference script. We can keep this version internal for our analysis

Reply:
Yeah, no benchmarking code in the final code, I agree.